### PR TITLE
Fix typo doc docker-stack

### PR DESCRIPTION
**Pull request by raphaelauv**
Fix a typo on doc , the _AIRFLOW_WWW_USER_PASSWORD_CMD is repeated two times

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: docs/docker-stack/entrypoint.rst [hunk 1]
```diff
@@ -131,7 +131,7 @@ The entrypoint can also create webserver user automatically when you enter it. y
 ``_AIRFLOW_WWW_USER_CREATE`` to a non-empty value in order to do that. This is not intended for
 production, it is only useful if you would like to run a quick test with the production image.
 You need to pass at least password to create such user via ``_AIRFLOW_WWW_USER_PASSWORD_CMD`` or
-``_AIRFLOW_WWW_USER_PASSWORD_CMD`` similarly like for other ``*_CMD`` variables, the content of
+``_AIRFLOW_WWW_USER_PASSWORD`` similarly like for other ``*_CMD`` variables, the content of
```
Comment (XD-DENG) [inline docs/docker-stack/entrypoint.rst]: Thanks @raphaelauv .

I agree this is a typo. But by reading through the sentence, I think the order of ``_AIRFLOW_WWW_USER_PASSWORD_CMD`` and ``_AIRFLOW_WWW_USER_PASSWORD`` should be swapped in your change.

**Review by XD-DENG — CHANGES_REQUESTED** [review]
Thanks @raphaelauv . 

By reading through the sentence, I think the order of

File: docs/docker-stack/entrypoint.rst (hunk 1 above)
Comment (raphaelauv) [reply docs/docker-stack/entrypoint.rst]: done , thank you

**Review by XD-DENG — APPROVED** [review]

File: docs/docker-stack/entrypoint.rst (hunk 1 above)
Comment (XD-DENG) [reply docs/docker-stack/entrypoint.rst]: Merci :)

**Pull request merged**